packages/client/src/components/measurements/MeasurementStatisticsBelowChart.re

```diff
@@ -0,0 +1,131 @@
+[@react.component]  
+let make =  
+    (
+        ~cdf: ForetoldComponents.Types.Dist.t, 
```

Comment:
In general, I'd recommend against using the types from ForetoldComponents in this app, unless we're using them in order to interact with these components directly.